http-clients/url-connection-client/src/main/java/software/amazon/awssdk/http/urlconnection/UrlConnectionHttpClient.java

```diff
@@ -228,6 +244,90 @@ public HttpExecuteResponse call() throws IOException {
                                       .build();
         }
 
+        private Optional<OutputStream> tryGetOutputStream() {
+            return getAndHandle100Bug(() -> invokeSafely(connection::getOutputStream), false);
+        }
+
+        private Optional<InputStream> tryGetInputStream() {
+            return getAndHandle100Bug(() -> invokeSafely(connection::getInputStream), true);
+        }
+
+        private Optional<InputStream> tryGetErrorStream() {
+            InputStream result = invokeSafely(connection::getErrorStream);
+            if (result == null && expect100BugEncountered) {
+                log.debug(() -> "The response payload has been dropped because of a limitation of the JDK's URL Connection "
```

Comment:
Is there anyway we can trace/correlate this to a given request? Should we include the status line?

Reply:
Since it's a sync client, they can use the thread name?